src/graphql/account.rs

```diff
@@ -418,6 +418,60 @@ impl AccountMutation {
         update_jwt_expires_in(expires_in)?;
         Ok(time)
     }
+
+    /// Retrieves the user's language selection by username.
+    #[graphql(guard = "RoleGuard::new(super::Role::SystemAdministrator)
+        .or(RoleGuard::new(super::Role::SecurityAdministrator))
+        .or(RoleGuard::new(super::Role::SecurityManager))
+        .or(RoleGuard::new(super::Role::SecurityMonitor))")]
+    async fn language(&self, ctx: &Context<'_>, username: String) -> Result<Option<String>> {
+        let store = crate::graphql::get_store(ctx).await?;
+        let map = store.account_map();
+
+        let account = map
+            .get(&username)?
+            .ok_or_else::<async_graphql::Error, _>(|| "Account not found".into())?;
+
+        Ok(account.language)
```

Comment:
These two lines can be combined into one if you use `map`.

Reply:
I used `map` to combine these lines.